addons/website/static/src/components/configurator/configurator.js

```diff
@@ -617,6 +618,30 @@ async function applyConfigurator(self, themeName) {
         self.env.router.navigate({to: 'CONFIGURATOR_PALETTE_SELECTION_SCREEN'});
         return;
     }
+
+    async function attemptConfiguratorApply(data, retryCount = 0) {
+        try {
+            const resp = await self.rpc({
+                model: 'website',
+                method: 'configurator_apply',
+                kwargs: data,
+            });
+            window.sessionStorage.removeItem(SESSION_STORAGE_ITEM_NAME);
+            window.location = resp.url;
```

Comment:
Wouldn't it be better to keep those lines outside of this method and make this method return the rpc?
```suggestion
```

Reply:
Nitpicking but I would prefer that too. Also in general I like returning an unresolved Promise after a redirect, so that I stop any JS code execution once the reload is asked (depends on how the caller is implemented of course).